packages/framework/fluid-static/tsconfig.json

```diff
@@ -1,13 +1,10 @@
 {
-	"extends": [
-		"../../../common/build/build-common/tsconfig.base.json",
-		"../../../common/build/build-common/tsconfig.cjs.json",
-	],
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
 	"include": ["src/**/*"],
 	"exclude": ["src/test/**/*"],
 	"compilerOptions": {
 		"rootDir": "./src",
-		"outDir": "./dist",
+		"outDir": "./lib",
 		"types": ["node"],
```

Comment:
pre-existing but this does not look right. I don't think fluid-static should import node types.

Reply:
Removed, thx.